examples/serverless-v10.yml

```diff
@@ -33,7 +33,8 @@ fauna:
 
   functions:
     FuncV10:
-      body: _ => 1
+      body: x => 1 + x
+      signature: "(x: Number) => Number"
```

Comment:
note that this is also a valid signature:
```
signature: "(Number) => Number"
```

in case you'd like to avoid duplicating parameter names.